crowd_anki/anki/hook_vendor.py

```diff
@@ -4,17 +4,13 @@
 from ..anki.adapters.hook_manager import AnkiHookManager
 from ..export.anki_exporter_wrapper import exporters_hook
 from ..history.archiver_vendor import ArchiverVendor
-from ..utils.config import AUTOMATED_SNAPSHOT
 
 
 @dataclass
 class HookVendor:
     window: Any
+    config: object
```

Comment:
should be of a type of the class you've created (here and elsewhere)

Reply:
I left it as an object in the places which don't actually use it, they only pass it on. But I can change that if you prefer.

Reply:
as we need to specify some type, we might as well specify a correct one here ;)

Reply:
Done